tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1,8 +1,12 @@
 const { client } = require('nightwatch-api')
 const assert = require('assert')
-const { Given, When, Then } = require('cucumber')
+const { Given, When, Then, Before } = require('cucumber')
 const webdav = require('../helpers/webdavHelper')
-let deletedElementsTable = []
+let deletedElements
+
+Before(() => {
+  deletedElements = []
```

Comment:
this needs to be reset so that the test run doesn't get affected by the previous scenario.